### DISCUSSION TITLE
Attaching hard drive images to SCSI IDs on startup

### DISCUSSION THREAD

**Discussion by MaCJaX** [akuker/RASCSI#1395] [General]
I am needing to attached hard drive images to SCSI IDs on reboot. I seen there is a way to edit the rascsi.service config file but that doesn't seem to exist anymore. Is there a new config name I should be looking for or another way to attach images on startup? Thanks !

**Comment by rdmark** [comment]
If you read the piscsi manual in the wiki, we have a section on this topic here: https://github.com/PiSCSI/piscsi/wiki/Drive-Setup#presenting-scsi-disks-when-the-piscsi-service-starts

Please let us know if you're still stuck after reading this!

**Reply by MaCJaX** [reply]
Thank you! I was looking for something like this but couldn't locate it.
